masu/processor/orchestrator.py

```diff
@@ -87,11 +88,13 @@ def prepare(self):
         """
         async_result = None
         for account in self._accounts:
-            LOG.info('Calling get_report_files with account: %s', account)
-            async_result = get_report_files.delay(**account)
-            LOG.info('Download queued - customer: %s, Task ID: %s',
-                     account.get('customer_name'),
-                     str(async_result))
+            provider_status = ProviderStatusAccessor(account.get('provider_uuid'))
+            if provider_status.is_valid() and not provider_status.is_backing_off():
```

Comment:
Back to the purpose of the `WARNING` state.  I think we could accomplish the same thing with having only the retry count.